### PR TITLE
Fix 80971: func_get_args() didnt work when using named args

### PR DESCRIPTION
**Pull request by sy-records**

### CODE REVIEW

**Comment by Girgias** [comment]
I'm not sure I understand what the note is meant to convey, is it that these two functions don't "see" named arguments?

Sidenote: can you rebase on top of master instead of merging master into the branch? Makes for easier rebase+fast forward merges

**Comment by sy-records** [comment]
> I'm not sure I understand what the note is meant to convey, is it that these two functions don't "see" named arguments?

yeah, 

> Sidenote: can you rebase on top of master instead of merging master into the branch? Makes for easier rebase+fast forward merges

done.

File: language-snippets.ent [hunk 1]
```diff
@@ -67,6 +67,13 @@ current scope to determine parameter details, it cannot be used as a
 function parameter in versions prior to 5.3.0. If this value must be passed, the results should be assigned
 to a variable, and that variable should be passed.</para></note>'>
 
+<!ENTITY note.func-named-params '<note xmlns="http://docbook.org/ns/docbook"><para>As of PHP 8, The func_*() family of
+functions is intended to be mostly transparent with regard to named arguments,
+by treating the arguments as if they were all passed positionally,
+and missing arguments were replaced with their defaults.
```
Comment (Girgias) [inline language-snippets.ent:73]:
```suggestion
and missing arguments are replaced with their defaults.
```
Or this talking about behaviour prior to PHP 8.0?

File: language-snippets.ent [hunk 2]
```diff
@@ -67,6 +67,13 @@ current scope to determine parameter details, it cannot be used as a
...
+This function oblivious to the collection of unknown named arguments by variadics.
+Collected unknown named arguments can only be accessed through the variadic parameter.</para></note>'>
```
Comment (Girgias) [inline language-snippets.ent:75]:
```suggestion
This function is oblivious to the collection of unknown named variadic arguments.
Unknown named arguments which are collected can only be accessed through the variadic parameter.</para></note>'>
```
Maybe is better phrasing, @afilina @cmb69 ?

Comment (afilina) [reply language-snippets.ent:75]: Maybe this is a bit better, I'm not sure.

```suggestion
This function ignores the collection of unknown named arguments by variadics.
Collected unknown named arguments can only be accessed through the variadic parameter.</para></note>'>
```

File: language-snippets.ent [hunk 3]
```diff
@@ -71,8 +71,8 @@ to a variable, and that variable should be passed.</para></note>'>
 functions is intended to be mostly transparent with regard to named arguments,
 by treating the arguments as if they were all passed positionally,
 and missing arguments are replaced with their defaults.
-This function oblivious to the collection of unknown named arguments by variadics.
-Collected unknown named arguments can only be accessed through the variadic parameter.</para></note>'>
+This function ignores oblivious to the collection of unknown named variadic arguments.
```
Comment (afilina) [inline language-snippets.ent:74]: It's either "ignores" or "oblivious to", but not both.

File: language-snippets.ent [hunk 4]
```diff
@@ -67,6 +67,13 @@ current scope to determine parameter details, it cannot be used as a
 function parameter in versions prior to 5.3.0. If this value must be passed, the results should be assigned
 to a variable, and that variable should be passed.</para></note>'>
 
+<!ENTITY note.func-named-params '<note xmlns="http://docbook.org/ns/docbook"><para>As of PHP 8, The func_*() family of
```
Comment (cmb69) [inline language-snippets.ent:70]:
```suggestion
<!ENTITY note.func-named-params '<note xmlns="http://docbook.org/ns/docbook"><para>As of PHP 8.0.0, the func_*() family of
```

**Review by cmb69 — COMMENTED** [review]
Except for the two outstanding comments, this looks good to me.

File: language-snippets.ent (hunk 3 above)
Comment (sy-records) [reply language-snippets.ent]: Thanks, I've updated

**Review by cmb69 — APPROVED** [review]
Thank you!

**Pull request merged**